Drivers/ProductAttributesPartDriver.cs

```diff
@@ -87,7 +91,7 @@ public class ProductAttributesPartDriver : ContentPartDriver<ProductAttributesPa
         protected override DriverResult Editor(ProductAttributesPart part, IUpdateModel updater, dynamic shapeHelper) {
             var editViewModel = new ProductAttributesEditViewModel();
             if (updater.TryUpdateModel(editViewModel, Prefix, null, null)) {
-                part.AttributeIds = editViewModel.AttributeIds;
+                part.AttributeIds = _attributeService.GetAttributes(editViewModel.AttributeIds).Select(pap => pap.Id);

```

Comment:
I'm not sure I understand why we need to do this. Can you explain why this can be different?